src/test/java/core/test/StructureTest.java

```diff
@@ -16,7 +16,7 @@
 
 public class StructureTest {
   private static final List<String> figureClassNames = List
-      .of("Circle", "Square", "IsoscelesTrapezoid", "Rectangle", "RightTriangle");
+      .of("Circle", "core.basesyntax.Square", "IsoscelesTrapezoid", "Rectangle", "RightTriangle");
```

Comment:
Please return this line to initial value to fix the build and proceed with review